src/mode/mysql_highlight_rules.js

```diff
@@ -5,18 +5,21 @@ var TextHighlightRules = require("./text_highlight_rules").TextHighlightRules;
+    var builtins = "rank|coalesce|ifnull|isnull|nvl";
     var variable = "charset|clear|connect|edit|ego|exit|go|help|nopager|notee|nowarning|pager|print|prompt|quit|rehash|source|status|system|tee";
-
+    var datetimeFunctions = 'adddate|addtime|convert_tz|curdate|current_date|current_time|current_timestamp|curtime|date|date_add|date_format|date_sub|datediff|day|dayname|dayofmonth|dayofweek|dayofyear|extract|from_days|from_unixtime|get_format|hour|last_day|localtime|localtimestamp|makedate|maketime|microsecond|minute|month|monthname|now|period_add|period_diff|quarter|sec_to_time|second|str_to_date|subdate|subtime|sysdate|time|time_format|time_to_sec|timediff|timestamp|timestampadd|timestampdiff|to_days|to_seconds|unix_timestamp|utc_date|utc_time|utc_timestamp|week|weekday|weekofyear|year|yearweek';
+    var encryptionFunctions = 'aes_decrypt|aes_encrypt|compress|md|random_bytes|sha|sha|statement_digest|statement_digest_text|uncompress|uncompressed_length|validate_password_strength';
+    var mathFunctions = 'abs|acos|asin|atan|atan|ceil|ceiling|conv|cos|cot|crc|degrees|div|exp|floor|ln|log|log|log|mod|pi|pow|power|radians|rand|round|sign|sin|sqrt|tan|truncate';
```

Comment:
A couple of functions are repeated here (e.g. `log`), looking at the math functions on the [mysql page](https://dev.mysql.com/doc/refman/8.0/en/mathematical-functions.html) it seems that the repetition comes from the numbers being chopped of from `log`, `log10`, and `log2`?

Reply:
Ups, my fault. Fixed and re-committed